raft/test/raft_test.cpp

```diff
@@ -585,15 +584,15 @@ namespace bzn
         resp.clear();
 
         // send invalid entry. peer should return false and not move back past commit index
-        msg = bzn::create_append_entries_request(TEST_NODE_UUID, 2, 0, 0, 0, 0, entry);
+        msg = bzn::create_append_entries_request(TEST_NODE_UUID, 2, 1, 0, 0, 0, entry);
         mh(msg, mock_session);
         ASSERT_FALSE(resp["data"]["success"].asBool());
         EXPECT_EQ(resp["data"]["matchIndex"].asUInt(), Json::UInt(1));
 
         resp.clear();
 
         // put back append entries - bad append entry
-        msg = bzn::create_append_entries_request(TEST_NODE_UUID, 2, 1, 0, 0, 2, entry);
+        msg = bzn::create_append_entries_request(TEST_NODE_UUID, 2, 2, 0, 0, 2, entry);
```

Comment:
I'm not sure the intent of this message, but I _think_ it should have the same prev_index and commit_index as the last rejected message, and be rejected because its prev_term is wrong. To that end, the previous invalid message should have valid prev_term.